### PR TITLE
SW-171: OK, next try: Don't do any passive logins during boot phase (and 10 secs after), also wait 3 secs before actually performing the passive login.

### PR DESCRIPTION
**Pull request by andywerner**

### CODE REVIEW

File: octoprint_mrbeam/static/js/mrbeam.js [hunk 1]
```diff
@@ -440,10 +443,17 @@ $(function () {
         };
 
         self._handle_session_expired = function (triggerUrl) {
+            // don't do this during boot time.
+            if (
+                self.isCurtainOpened <= 0 &&
+                self.isCurtainOpened + 10 * 1000 > Date.now()
```
Comment (khaledsherkawi) [inline octoprint_mrbeam/static/js/mrbeam.js:449]: is 10 seconds here an assumption for how long the booting takes??

**Review by khaledsherkawi — DISMISSED** [review]

File: octoprint_mrbeam/static/js/mrbeam.js (hunk 1 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/static/js/mrbeam.js:449]: I'm not sure the "less than" services any purpose here. please clarify.

Comment (andywerner) [reply octoprint_mrbeam/static/js/mrbeam.js:449]: 1. It is the moment the "onCurtainOpened" event fired + 10 seconds. 
2. self.isCurtainOpened is initialised to 0 (I see it as a timestamp). It remains zero until the "onCurtainOpened" event fires, then it is set ti the current timestamp. The "less than" is "just to be on the save side" incase someone initialises or sets it's value to -1.

**Review by khaledsherkawi — APPROVED** [review]

File: octoprint_mrbeam/static/js/mrbeam.js (hunk 1 above)
Comment (amrsoll) [reply octoprint_mrbeam/static/js/mrbeam.js:449]: wouldn't it be easier to check if the curtain is still active or displayed?

**Pull request merged**